apps/comps/e2e/tests/admin.test.ts

```diff
@@ -253,115 +236,105 @@ describe("Admin API", () => {
-    await adminClient.loginAsAdmin(adminApiKey);
-
     // Register a new user with no agent
     const userName = `Test User ${Date.now()}`;
     const userWalletAddress = generateRandomEthAddress();
-    const userResult = (await adminClient.registerUser({
+    const userResult = await authorizedAdminClient.users.register({
       walletAddress: userWalletAddress,
       name: userName,
       email: `${userName.toLowerCase().replace(/\s+/g, "-")}@test.com`,
-    })) as UserRegistrationResponse;
+    });
     expect(userResult.success).toBe(true);
     expect(userResult.user).toBeDefined();
     const userId = userResult.user.id;
     expect(userId).toBeDefined();
 
     // Register the agent with an invalid user ID
     const randomUuid = "ca00c26a-f610-477a-a472-bdaca866d789";
-    const agentResult = (await adminClient.registerAgent({
-      user: {
-        id: randomUuid,
-      },
-      agent: {
-        name: "Test Agent",
-        description: "A test trading agent",
-        walletAddress: generateRandomEthAddress(),
-      },
-    })) as ErrorResponse;
-    expect(agentResult.success).toBe(false);
-    expect(agentResult.error).toContain("does not exist");
+    await assertRpcError(
+      authorizedAdminClient.agents.create({
+        user: {
+          id: randomUuid,
+        },
+        agent: {
+          name: "Test Agent",
+          description: "A test trading agent",
+          walletAddress: generateRandomEthAddress(),
+        },
+      }),
+      "NOT_FOUND",
+    );
 
     // Register the agent with an invalid user ID and wallet address
     const randomWalletAddress = generateRandomEthAddress();
-    const agentResult2 = (await adminClient.registerAgent({
-      user: {
-        walletAddress: randomWalletAddress,
-      },
-      agent: {
-        name: "Test Agent",
-        description: "A test trading agent",
-        walletAddress: generateRandomEthAddress(),
-      },
-    })) as ErrorResponse;
-    expect(agentResult2.success).toBe(false);
-    expect(agentResult2.error).toContain("does not exist");
+    await assertRpcError(
+      authorizedAdminClient.agents.create({
+        user: {
+          walletAddress: randomWalletAddress,
+        },
+        agent: {
+          name: "Test Agent",
+          description: "A test trading agent",
+          walletAddress: generateRandomEthAddress(),
+        },
+      }),
+      "NOT_FOUND",
+    );
 
     // Register the agent with both a user ID and a user wallet address
-    const agentResult3 = (await adminClient.registerAgent({
-      user: {
-        id: randomUuid,
-        walletAddress: userWalletAddress,
-      },
-      agent: {
-        name: "Test Agent",
-        description: "A test trading agent",
-        walletAddress: generateRandomEthAddress(),
-      },
-    })) as ErrorResponse;
-    expect(agentResult3.success).toBe(false);
-    expect(agentResult3.error).toContain(
-      "must provide either user ID or user wallet address",
```

Comment:
not a huge deal, but we lost the checking of the specific error message here, might be nice to keep that if easy to do so (here and elsewhere in this test file)